types/tipset.go

```diff
@@ -120,11 +124,11 @@ func (ts TipSet) MinTicket() (Ticket, error) {
 	if len(ts.blocks) == 0 {
 		return Ticket{}, errUndefTipSet
 	}
-	// TODO #2223 fix this to properly handle multiple tickts per block
-	if len(ts.blocks[0].Tickets) == 0 {
+	numTickets := len(ts.blocks[0].Tickets)
+	if numTickets == 0 {
```

Comment:
We can now catch this in the constructor and prevent the creation of such a tipset (though that doesn't win us much since there's still an error return value here).

Reply:
Strong preference for the former, reasoning about the errors these methods return is easier (imo) when there is only one type of error.

Reply:
SGTM.  Since we already check for this in the constructor I'll just kill this extraneous check